.circleci/config.yml

```diff
@@ -351,6 +351,15 @@ jobs:
           command: | # Justfiles are not shell scripts and should not be checked with shellcheck
             (cd src/improvements/script && shellcheck -x *)
 
+scheduled:
+  triggers:
+    - schedule: # Running CI on main every 4 hours
+        cron: "0 */4 * * *"
+        filters:
+          branches:
+            only:
+              - main
```

Comment:
Does this work, where does this get applied to the main workflow? In the monorepo we use the e.g. `build_four_hours` trigger:
- config: https://app.circleci.com/settings/project/github/ethereum-optimism/optimism/triggers?return-to=https://app.circleci.com/pipelines/github/ethereum-optimism/optimism
- applying it: https://github.com/ethereum-optimism/optimism/blob/420be9caa771f7136e65b1bbe47c383892b7b595/.circleci/config.yml#L1791

Reply:
Interesting I haven't used triggers like that in circleci before. Though I have used triggers this way via the yml file itself (security-tools).

Reply:
I still don't understand how this gets applied to the main workflow?

Reply:
I've changed the approach to match what we do in the monorepo.

Reply:
https://github.com/ethereum-optimism/superchain-ops/pull/558/commits/2e68695e36a06c70297aaf194228d6b433e13979 and see [here](https://app.circleci.com/settings/project/gh/ethereum-optimism/superchain-ops/triggers?return-to=https%3A%2F%2Fapp.circleci.com%2Fpipelines%2Fgh%2Fethereum-optimism%2Fsuperchain-ops).